ironfish-cli/src/commands/wallet/transactions.ts

```diff
@@ -111,4 +114,42 @@ export class TransactionsCommand extends IronfishCommand {
       showHeader = false
     }
   }
+
+  getTransactionRow(transaction: GetAccountTransactionsResponse): TransactionRow {
+    const assetId = Asset.nativeId().toString('hex')
+
+    const nativeAssetBalanceDelta = transaction.assetBalanceDeltas.find(
+      (d) => d.assetId === assetId,
+    )
+
+    let amount = BigInt(nativeAssetBalanceDelta?.delta ?? '0')
+
+    let feePaid = BigInt(transaction.fee)
+
+    if (transaction.type !== TransactionType.SEND) {
+      feePaid = 0n
+    } else {
+      amount += feePaid
+    }
```

Comment:
Can we also adjust the amount for miners transactions too?

Reply:
This is what I get for miner's fees testing locally: 
<img width="1302" alt="image" src="https://user-images.githubusercontent.com/57735705/214161200-76c61828-ef4a-4fa6-85f8-2d9427167c01.png">

Should it be adjusted differently?